go.mod

```diff
@@ -32,9 +32,10 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/secure-systems-lab/go-securesystemslib v0.8.0
 	github.com/sigstore/fulcio v1.5.1
-	github.com/sigstore/protobuf-specs v0.3.0
+	github.com/sigstore/protobuf-specs v0.3.2
 	github.com/sigstore/rekor v1.3.6
 	github.com/sigstore/sigstore v1.8.7
+	github.com/sigstore/sigstore-go v0.4.1-0.20240717174219-8554eb6de5ac
```

Comment:
To make use of https://github.com/sigstore/sigstore-go/pull/235. Once https://github.com/sigstore/sigstore-go/pull/236 also lands we should consider cutting a numbered release of sigstore-go to include these two changes.

Reply:
Merged - shall we merge all the open dependabot PRs and then cut a new release?